packages/cells/src/cells/multi-select-cell.tsx

```diff
@@ -0,0 +1,514 @@
+                        ? `Create "${input.inputValue}"`
+                        : undefined;
+                }}
+                menuIsOpen={cell.readonly ? false : menuOpen}
+                onMenuOpen={() => setMenuOpen(true)}
+                onMenuClose={() => setMenuOpen(false)}
+                value={resolveValues(value, options, allowDuplicates)}
+                onKeyDown={cell.readonly ? undefined : handleKeyDown}
+                menuPlacement={"auto"}
+                menuPortalTarget={document.getElementById("portal")}
+                autoFocus={true}
+                openMenuOnFocus={true}
+                openMenuOnClick={true}
+                closeMenuOnSelect={true}
+                backspaceRemovesValue={true}
+                escapeClearsValue={false}
+                styles={colorStyles}
+                components={{
+                    DropdownIndicator: () => null,
+                    IndicatorSeparator: () => null,
+                    Menu: props => {
+                        if (menuDisabled) {
+                            return null;
+                        }
+                        return (
+                            <PortalWrap>
+                                <CustomMenu className={"click-outside-ignore"} {...props} />
+                            </PortalWrap>
+                        );
+                    },
+                }}
+                onChange={async e => {
+                    if (e === null) {
+                        return;
+                    }
+                    submitValues(e.map(x => x.value));
+                }}
+            />
+        </Wrap>
+    );
+};
+
+const renderer: CustomRenderer<MultiSelectCell> = {
+    kind: GridCellKind.Custom,
+    isMatch: (c): c is MultiSelectCell => (c.data as any).kind === "multi-select-cell",
+    draw: (args, cell) => {
+        const { ctx, theme, rect, highlighted } = args;
+        const { values, options: optionsIn } = cell.data;
+
+        if (values === undefined || values === null) {
+            return true;
+        }
+
+        const options = prepareOptions(optionsIn ?? []);
+
+        const drawArea: Rectangle = {
+            x: rect.x + theme.cellHorizontalPadding,
+            y: rect.y + theme.cellVerticalPadding,
+            width: rect.width - 2 * theme.cellHorizontalPadding,
+            height: rect.height - 2 * theme.cellVerticalPadding,
+        };
+        const rows = Math.max(1, Math.floor(drawArea.height / (BUBBLE_HEIGHT + BUBBLE_PADDING)));
+
+        let { x } = drawArea;
+        let row = 1;
+
+        let y =
+            rows === 1
+                ? drawArea.y + (drawArea.height - BUBBLE_HEIGHT) / 2
+                : drawArea.y + (drawArea.height - rows * BUBBLE_HEIGHT - (rows - 1) * BUBBLE_PADDING) / 2;
+        for (const value of values) {
+            const matchedOption = options.find(t => t.value === value);
+            const colorHex = matchedOption?.color
```

Comment:
Why does this need to be hex? It seems like it could be any css color string?

Reply:
No need for this, removed it 👍